process/outportBlockConverter.go

```diff
@@ -13,58 +14,50 @@ import (
 	"github.com/multiversx/mx-chain-ws-connector-template-go/data"
 )
 
-var (
-	protoMarshaller = &marshal.GogoProtoMarshalizer{}
+type outportBlockConverter struct {
+	protoMarshaller *marshal.GogoProtoMarshalizer
```

Comment:
we can have a small wrapper component for standard proto (which implements our `marshal.Marshaller` interface) and inject it here also as a variable

Reply:
and `protoMarshaller` can be reanamed to `gogoProtoMarshaller` or something similar in order to better differentiate them